build.gradle

```diff
@@ -151,6 +151,7 @@ configure(javaProjects) {
         all*.exclude module: "jcl-over-slf4j"
         all*.exclude module: "log4j-over-slf4j"
         all*.exclude module: "slf4j-jdk14"
+        all*.exclude group: "ch.qos.logback", module: "logback-classic"
```

Comment:
if you're taking logback out what does the default logger use now? log4j?

Reply:
Yes, I thought it was always log4j.